packages/ui-extensions/src/surfaces/admin/components/Table/Table.doc.ts

```diff
@@ -132,7 +131,7 @@ const data: AdminReferenceEntityTemplateSchema = {
           },
           {
             description:
-              'Demonstrates a table with pagination controls, showing how to add previous and next page functionality to handle large datasets.',
+              'Add pagination controls for navigating large datasets.',
```

Comment:
is there value in adding example logic to show pagination? Or maybe we want to leave that up to the individual consumer, and we just want to show the available UI?

https://github.com/user-attachments/assets/c615ebb1-96a2-4581-8416-16705eb6e7f4